dbux-data/src/impl/indexes/SimpleDataNodesIndex.js

```diff
@@ -0,0 +1,24 @@
+import CollectionIndex from '../../indexes/CollectionIndex';
+import RuntimeDataProvider from '../../RuntimeDataProvider';
+
+/** @typedef {import('@dbux/common/src/types/DataNode').default} DataNode */
+
+/** @extends {CollectionIndex<DataNode>} */
+export default class SimpleDataNodesIndex extends CollectionIndex {
```

Comment:
@MiccWan 
rename `simple` to `primtive` maybe? That is the predominant terminology we use to refer to these types of values